arkouda/pdarrayclass.py

```diff
@@ -137,7 +187,31 @@ def binop(self, other : object, op : str):
 
     # reverse binary operators
     # pdarray binop pdarray: taken care of by binop function
-    def r_binop(self, other, op):
+    def r_binop(self, other : 'pdarray', op : str) -> 'pdarray':
```

Comment:
`r_binop` is not suppose to be user facing. maybe it should get underscores prepended to the name?

Reply:
Could not disagree less! 👍

Reply:
fixed!